packages/app/src/context-menu.ts

```diff
@@ -113,6 +113,23 @@ export default class ContextMenu extends EventEmitter {
             }
           },
         },
+        {
+          id: 'copyImage',
+          label: 'Copy Image to Clipboard',
+          click() {
+            // Fetch the image and copy to clipboard
+            fetch(props.srcURL)
+              .then(response => response.arrayBuffer())
+              .then(buffer => {
+                const { nativeImage } = require('electron');
```

Comment:
Move this `require` in the top of the file in the already existing `electron` import.